docs/conf.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import os
+import sys
+from datetime import datetime
+
+
+sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(1, os.path.abspath('../src/twister2'))
+
+import twister2  # noqa: E402
+
+
+# -- General configuration ---------------------------------------------
+
+# needs_sphinx = '1.0'
+
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx_autodoc_typehints',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.intersphinx',
+    'sphinx_rtd_theme',
+    'm2r2',

```

Comment:
please let's not introduce an extension just because we want to pull a markdown file, let's just use rst or markdown everywhere but not both. Also, with a standard Sphinx setup, all docs should be under the same source folder.

Reply:
I changed README.md to reStructuredText format, and this package is no required anymore.